packages/jest-editor-support/src/__tests__/parsers/TypeScriptParser-test.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+let hasTypeScript = false;
+let typescriptParser = null;
+
+try {
+  typescriptParser = require('../../parsers/TypeScriptParser').typescriptParser;
+  hasTypeScript = true;
+} catch (e) {
+  // NO-OP, there are legit reasons for not having typescript
+  // inside the node_modules.
```

Comment:
Can we make typescript a devDependency of the top level package.json in Jest to make sure this test is always run?